index.php

```diff
@@ -1,9 +1,13 @@
 <?php
 $links = array(
+    'blog' => 'https://tcmaker.org/news',
     'cal' => 'https://tcmaker.org/calendar',
     'classes' => 'https://tcmaker.org/classes',
     'eb' => 'https://www.eventbrite.com/o/twin-cities-maker-440413464',
     'fb' => 'https://www.facebook.com/groups/TwinCitiesMaker/',
```

Comment:
Consider changing line to: if we will be linking to other facebook resources.
`    'fbgrp' => 'https://www.facebook.com/groups/TwinCitiesMaker/',`
if we will be linking to other facebook resources.

Reply:
Are there, in the handbook, other links to Facebook resources? If there aren't, then this issue is moot.